.bazelrc

```diff
@@ -8,7 +8,7 @@ import %workspace%/envoy.bazelrc
 
 # Overrides workspace_status_command
 build --workspace_status_command=tools/bazel_get_workspace_status
-
+build:remote --remote_timeout=7200
```

Comment:
What's this for?

Reply:
Without this RBE will timeout for wee8 build, because istio-testing is using different type of machine than upstream.